test/common/assertSnapshot.js

```diff
@@ -36,11 +36,9 @@ async function assertSnapshot(actual, filename = process.argv[1]) {
 }
 
 async function spawnAndAssert(filename, transform = (x) => x) {
```

Comment:
Can you please add a JSDoc comment for this function, not just documenting its signature but also explaining how test authors can generate new snapshots? Including how to write a custom transform function in case the test author wants to test part of the stack trace that would be excluded by default. (Or if that’s already documented somewhere else, maybe the JSDoc comment can include a link to where to find those instructions.)

Reply:
don